src/test/java/com/google/sps/SendVideoInfoTest.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.when;
+import com.google.sps.data.*;
+import java.util.*;
+import com.google.sps.servlets.SyncServlet;
+
+@RunWith(JUnit4.class)
+public final class SendVideoInfoTest {
+    
+    @Test
+    public void testCorrectVideoStringOutput(){
+        Room testRoom = Mockito.mock(Room.class);
```

Comment:
no need to Mock Room class, especially since it has the constructor available now.